mypy.ini

```diff
@@ -25,3 +25,6 @@ ignore_missing_imports = True
 
 [mypy-checkov.*]
 follow_imports = skip
+
+[mypy-semgrep.*]
+ignore_missing_imports = True
```

Comment:
I can see they don't want people to use it as a library, they have type hints, but didn't add the `py.typed` marker to the project